### DISCUSSION TITLE
能帮忙打包这个软件么: wucai 本地打包总是失败

### DISCUSSION THREAD

**Discussion by weqoocu** [tw93/Pake#578] [General]
https://marker.dotalk.cn/#/

**Comment by Tlntin** [comment]
试试在线打包？[wiki](https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89)
